app/lib/tufts/technical_metadata.rb

```diff
@@ -0,0 +1,84 @@
+module Tufts
+  # A class to generate an XML document that contains all the technical
+  # metadata that has been persisted in Fedora after characterization
+  # with FITS
+  #
+  # It does this by loading an ActiveFedora::Base object for a work
+  # and then iterating through the FileSets on the work. It then
+  # creates an Nokogiri::XML::Builder object with values from the
+  # FileSet member's characterization proxy
+  #
+  # @example
+  # tm = Tufts::TechnicalMetadata.new('zvcvcvc23')
+  # tm.to_s
+  #
+  class TechnicalMetadata
+    ##
+    # @!attribute base, builder [r]
+    # @!attribute file_set [rw]
+    attr_reader :base, :builder
+    ##
+    # @param work_id [String]
+    def initialize(work_id)
+      @base = ActiveFedora::Base.find(work_id)
```

Comment:
Can we just pass the object in? The Fedora round trips for these find calls are going to be expensive for large exports.